staging/src/k8s.io/kubectl/pkg/cmd/label/label.go

```diff
@@ -182,6 +182,9 @@ func (o *LabelOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []st
 	}
 	o.resources = resources
 	o.newLabels, o.removeLabels, err = parseLabels(labelArgs)
+	if err != nil {
```

Comment:
This one is fine, there is validation right after that steps that verifies that newLabels and removeLabels are bigger than 1.